tools/champ_blocs/cb_clean.xml

```diff
@@ -0,0 +1,67 @@
+<tool id="cb_cleaning" name="Clean data" version="@VERSION@" profile = "20.01">
+    <description>for dissimilarity calculations</description>
```

Comment:
This tool has no parameters, correct and I assume it is also not super slow. What do you think of including in the dissimilarity tool as option? Avoiding this extra tool at all.

Reply:
This would also eliminate passing around the RData object :)